README.md

```diff
@@ -148,9 +148,13 @@ Available global options:
 Available commands:
   with                     Specify options to enable
   without                  Specify options to disable
+  show                     Show command. (Support licenses, license and ghcs)
```

Comment:
This is not completely correct documentation. Now as we have not only `new` command we need to reorganize this section. For this `--help` command will be very helpful 😉 

So the command section should contain the only commands we have now: `new`, `show`. (Use `summon --help` to see this).
Then for each of this commands you can write what we have at the moment. (Use `summon COMMAND --help` to see).